File: root_easysetup/programs/VB/Samples/WS_sample/WSClient_sample/WSClientWin2_sample/UserControl3.vb
'**********************************************************************************
'* サンプル アプリ画面
'**********************************************************************************

'**********************************************************************************
'* クラス名        ：UserControl3
'* クラス日本語名  ：ユーザコントロール
'*
'* 作成日時        ：－
'* 作成者          ：sas 生技
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'*
'**********************************************************************************

Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms


Imports Touryo.Infrastructure.Business.RichClient.Asynchronous
Imports Touryo.Infrastructure.Business.RichClient.Presentation

Imports Touryo.Infrastructure.Framework.RichClient.Asynchronous
Imports Touryo.Infrastructure.Framework.RichClient.Presentation

Partial Public Class UserControl3
    Inherits UserControl
    Public Sub New()
        InitializeComponent()
    End Sub

    Protected Sub UOC_btnUCButton1_Click(ByVal rcFxEventArgs As RcFxEventArgs)
        System.Diagnostics.Debug.WriteLine("UOC_btnUCButton1_Click")
    End Sub

    Protected Sub UOC_pbxUCPictureBox1_Click(ByVal rcFxEventArgs As RcFxEventArgs)
        System.Diagnostics.Debug.WriteLine("UOC_pbxUCPictureBox1_Click")
    End Sub

    Protected Sub UOC_rbnUCRadioButton1_CheckedChanged(ByVal rcFxEventArgs As RcFxEventArgs)
        System.Diagnostics.Debug.WriteLine("UOC_rbnUCRadioButton1_CheckedChanged")
    End Sub

    Protected Sub UOC_cbxUCCheckBox1_CheckedChanged(ByVal rcFxEventArgs As RcFxEventArgs)
        System.Diagnostics.Debug.WriteLine("UOC_cbxUCCheckBox1_CheckedChanged")
    End Sub

    Protected Sub UOC_cbbUCComboBox1_SelectedIndexChanged(ByVal rcFxEventArgs As RcFxEventArgs)
        System.Diagnostics.Debug.WriteLine("UOC_cbbUCComboBox1_SelectedIndexChanged")
    End Sub

    Protected Sub UOC_lbxUCListBox1_SelectedIndexChanged(ByVal rcFxEventArgs As RcFxEventArgs)
        System.Diagnostics.Debug.WriteLine("UOC_lbxUCListBox1_SelectedIndexChanged")
    End Sub
End Class

File: root/programs/CS/Samples4NetCore/Legacy/WS_sample/WSClient_sample/WSClientWin2_sample/UserControl3.cs
//**********************************************************************************
//* Windows Forms用 Ｐ層 フレームワーク・テスト アプリ画面
//**********************************************************************************

// テスト用サンプルなので、必要に応じて流用 or 削除して下さい。

//**********************************************************************************
//* クラス名        ：UserControl3
//* クラス日本語名  ：UserControl3
//*
//* 作成日時        ：－
//* 作成者          ：生技
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//*
//**********************************************************************************

using System.Diagnostics;
using System.Windows.Forms;
using Touryo.Infrastructure.Framework.RichClient.Presentation;

namespace WSClientWin2_sample
{
    /// <summary>UserControl3</summary>
    public partial class UserControl3 : UserControl
    {
        /// <summary>constructor</summary>
        public UserControl3()
        {
            InitializeComponent();
        }

        /// <summary>UOC_btnUCButton1_Click</summary>
        /// <param name="rcFxEventArgs">RcFxEventArgs</param>
        protected void UOC_btnUCButton1_Click(RcFxEventArgs rcFxEventArgs)
        {
            Debug.WriteLine("UOC_btnUCButton1_Click");
        }

        /// <summary>UOC_pbxUCPictureBox1_Click</summary>
        /// <param name="rcFxEventArgs">RcFxEventArgs</param>
        protected void UOC_pbxUCPictureBox1_Click(RcFxEventArgs rcFxEventArgs)
        {
            Debug.WriteLine("UOC_pbxUCPictureBox1_Click");
        }

        /// <summary>UOC_rbnUCRadioButton1_CheckedChanged</summary>
        /// <param name="rcFxEventArgs">RcFxEventArgs</param>
        protected void UOC_rbnUCRadioButton1_CheckedChanged(RcFxEventArgs rcFxEventArgs)
        {
            Debug.WriteLine("UOC_rbnUCRadioButton1_CheckedChanged");
        }

        /// <summary>UOC_cbxUCCheckBox1_CheckedChanged</summary>
        /// <param name="rcFxEventArgs">RcFxEventArgs</param>
        protected void UOC_cbxUCCheckBox1_CheckedChanged(RcFxEventArgs rcFxEventArgs)
        {
            Debug.WriteLine("UOC_cbxUCCheckBox1_CheckedChanged");
        }

        /// <summary>UOC_cbbUCComboBox1_SelectedIndexChanged</summary>
        /// <param name="rcFxEventArgs">RcFxEventArgs</param>
        protected void UOC_cbbUCComboBox1_SelectedIndexChanged(RcFxEventArgs rcFxEventArgs)
        {
            Debug.WriteLine("UOC_cbbUCComboBox1_SelectedIndexChanged");
        }

        /// <summary>UOC_lbxUCListBox1_SelectedIndexChanged</summary>
        /// <param name="rcFxEventArgs">RcFxEventArgs</param>
        protected void UOC_lbxUCListBox1_SelectedIndexChanged(RcFxEventArgs rcFxEventArgs)
        {
            Debug.WriteLine("UOC_lbxUCListBox1_SelectedIndexChanged");
        }
    }
}
